quarkus/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -43,20 +45,17 @@ static class InputOptions {
     FileInputOptions fileOptions;
 
     static class StandardInputOptions {
-
       @CommandLine.Option(
-          names = {"-r", "--realm"},
-          paramLabel = "<realm>",
-          required = true,
-          description = "The name of a realm to bootstrap.")
-      List<String> realms;
+          names = {"-c", "--credentials"},
+          description =
+              "Principal credentials to bootstrap. If provided, must be a valid JSON array e.g. "
+                  + "[{\"realm\": \"my-realm\", \"principal\": \"root\", \"clientId\": \"polaris\", \"clientSecret\": \"p4ssw0rd\"}]")
+      String credentials;
 
       @CommandLine.Option(
-          names = {"-c", "--credential"},
-          paramLabel = "<realm,clientId,clientSecret>",
```

Comment:
What is the reason for removing the option for the user to specify plain text credentials as command line args?

Reply:
Refer to my comments and -1 on #633 for more context, but essentially I don't think this format captures the full range of ways you might wish to bootstrap. I also don't think we really want two syntaxes for bootstrapping.

With that being said, I'd like to understand why the [pre-Quarkus bootstrap behavior](https://github.com/apache/polaris/pull/461#issuecomment-2666608052) is now gone, since being able to bootstrap a realm without specifying credentials is a core part of why this heavier syntax makes sense.

Reply:
I believe the existing `-c` option is adequate for bootstrapping current Polaris Server and it is convenient for users.

I think we should keep it, but I'm open to altering its format as long as it is human-writable in shells without too much quoting.